.github/actions/save-logs/action.yaml

```diff
@@ -44,3 +44,4 @@ runs:
           /tmp/debug-*.log
           /tmp/uds-containerd-logs
           /tmp/k3d-uds-*.log
+          assessment-results.yaml
```

Comment:
I don't seem to see this file in the debug logs?
![image](https://github.com/defenseunicorns/uds-common/assets/3977569/5f31d251-3d70-433b-9246-6576cd4678e0)
https://github.com/defenseunicorns/uds-common/actions/runs/9669308850?pr=157

Reply:
great point - changing the task use to include writing that file to tmp. Was previously mis-named.